ndex.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<!-- <html> Question 1 part A, B
+    <head>
+        <script>
+            let userString = window.prompt("Enter your name")
+            window.alert(`Thank you ${userString}`);
+                </script>
+    </head>
+
+    <body>
+        <h1>Pop up String</h1>
+        <p id="page">
+             <script>
+                document.getElementById("page").innerHTML = "Your name is " + userString ;
+                </script>
+        </p>
+    </body>
+</html>
+
+<html> <!--Question 2 part A, B-->
+   <!-- <head>
+   </head>
+   <body>
+       <script>
+        let userNum = window.prompt("Enter a number");
+        if(userNum % 2===0){
+            window.alert("Your number is even")
+            document.getElementById("page").innerHTML = "Your number is Even " + userNum ;
+        } else{
+            window.alert("your number is not even")
+            document.getElementById("page").innerHTML = "Your number is not even " + userNum ;
+        } 
+        
+        </script>
+
+
+   </body> -->
+<!-- </html> --> -->
+
+<!-- <html>
+    <head>
+        <title> Double your number below! </title>
+    </head>
+    <body>
+        <h1>Double your number below!</h1>
+
+            Insert Num:<br>
+            <input id= "num" type="text" /> <br>
+            <button type="button" onclick="double()">Press me!</button>
+
+        
+        <script>
+            const double = () => {
+                let num = document.getElementById("num").value  
+                let newNum = num*2             
+                document.getElementById("num").value = "Your number is " + newNum
+                return window.alert(newNum)
+            }
+
+        </script>
+        <p id="page">
+
+        </p>
+    </body>
+</html> -->
+
+<!-- <html>
+    <head>
+        <title>Palindrome detector</title>
+    </head>
+    <body>
+        <h1>Palindrome!</h1>
+        Insert a word:
+        <input id="word" type="text" /> <br>
+        <button type="button" onclick="palindromeCheck()">Check here!</button>
+        <script>
+        const palindromeCheck = () => {
+            let str = document.getElementById("word").value
+            isPali = false
+            strReverse = str.split("").reverse()
```

Comment:
If you're going to take this approach it can be just simpler to reverse it and compare it to that same string.



const isPali = str => str === str.split('').reverse().join('')